android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -242,6 +250,31 @@ class AirshipPlugin : MethodCallHandler {
         editor.apply();
     }
 
+    private fun editAttributes(call: MethodCall, result: Result) {
+        val mutations = call.arguments as ArrayList<Map<String, Any?>>
+        this.applyAttributesOperations(UAirship.shared().channel.editAttributes(), mutations)
+        result.success(null)
+    }
+
+    private fun applyAttributesOperations(editor: AttributeEditor, operations: ArrayList<Map<String, Any?>>) {
+        for (i in 0 until operations.size) {
```

Comment:
this is not a critique, but since I am new to dart I did some googling and found that for looping over ArrayList you should also be able to say:

for (var operation in operations) {
}

and

operations.forEach((operation) {

)};

Reply:
yeah not sure why I did this. Fixed with reasonable kotlin alternative